server/src/recipe/recipe.controller.js

```diff
@@ -44,4 +44,20 @@ const createRecipe = async (req, res, next) => {
   }
 };
 
-export { listPublishedRecipes, listUserRecipes, createRecipe };
+const deleteRecipe = async (req, res, next) => {
+  const recipeId = await req.body.recipeId;
+
+  try {
+    const recipe = await Recipe.findOne({
+      where: { id: recipeId },
+    });
+    if (recipe) {
```

Comment:
you need to let the user know that they are trying to delete something that doesnt exist

```js
if(!recipe) return new HttpError(NOT_FOUND, not found message)
await recipe.destroy()...
```